spec/factories/locations.rb

```diff
@@ -0,0 +1,10 @@
+FactoryBot.define do
+  factory :location do
+    association :locatable
```

Comment:
I'm not 100% sure on whether this is necessary. The test suite passes with or without.